experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import { Container } from "@fluidframework/container-loader";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import { IEvent } from "@fluidframework/common-definitions";
+import { IAudience } from "@fluidframework/container-definitions";
+import { LoadableObjectClass, LoadableObjectRecord } from "./types";
+import { RootDataObject } from "./rootDataObject";
+interface IFluidContainerEvents extends IEvent {
```

Comment:
nit: perhaps we can add a newline after imports

Reply:
```suggestion

interface IFluidContainerEvents extends IEvent {
```